frontend/src/routes/Router.tsx

```diff
@@ -16,7 +16,7 @@ const routes = createRoutesFromElements(
     {/* TODO: Different navbar/layout for the unauthenticated. */}
     {/* Unauthenticated stuff: */}
```

Comment:
This todo is about the navbar right corner and the AuthenticationGuard element below